py/kubeflow/tf_operator/util.py

```diff
@@ -18,7 +18,7 @@
 import google.auth.transport.requests
 import requests
 import yaml
-from googleapiclient import errors
+#from googleapiclient import errors
```

Comment:
does the file still have reference to `errors`?

Reply:
It is used in create_cluster and delete cluster for gke. It should not be referenced now? 

https://github.com/kubeflow/tf-operator/blob/0c41b273120edb59ef21d2ef77560412a60394c2/py/kubeflow/tf_operator/util.py#L229

Reply:
Sounds good, we can create some clean up tasks later.